Jenkinsfile

```diff
@@ -30,6 +30,10 @@ setup_mxnet_gpu = """
     export MXNET_CUDNN_AUTOTUNE_DEFAULT=0
 """
 
+setup_torch_gpu = """
+    python3 -m pip install torch==1.7.1+cu101 torchvision==0.8.2+cu101 -f https://download.pytorch.org/whl/torch_stable.html
```

Comment:
Is this necessary for the default install of AG to use TIMM? How does this differ from pip install torch as in Tabular?

Our CI should probably not be installing a different version of Torch than is installed by users unless we document it in our installation instructions.

Reply:
timm-clean is a pure python package with no dependency at all so it's safe. Regarding torch installation, it's okay to use single `pip install torch` but how do you make sure that pip will pick up the right version? AFAIK pip install torch will require the specific cuda version otherwise you have to use conda install with the proper cuda version.


For example, the default version of cuda is 10.2 if you install using `pip install torch`  per instruction here(https://pytorch.org/get-started/locally/)